### ISSUE TITLE
Absorb dark energy

### ISSUE THREAD

**Issue by just4omp** [kajdaq/stunning-chainsaw#1]
When I use Absorb Dark Energy I select filter Old - Unique and deselect Magic in game, in script i select all Old - Unique. First time it absorb normally, but when script absorb 2nd time, it deselect Magic and Rare, on 3rd select them again.  Then sometimes script can deselect first two (old and normal), and select them again in next absorb. 

May be its possible to choose use in-game filter, or use script filter? Dont think that script filter really needed, its ok just function absorb or not items.

I use LD player, 1280*720, Ankulua 8.6.1, Screenshot by - Media Projection

**Comment by kajdaq** [comment]
This is the next priority.  Temporary solution in next release will be to disable the filter checking/selection function and have the player manually select the filters prior to turning on the bot.  A temporary release to that effect will be out this weekend hopefully.  Still familiarizing myself with the code.

**Comment by just4omp** [comment]
Thanks you very much!

**Comment by kajdaq** [comment]
I'll close the issue at the new release.

**Comment by just4omp** [comment]
Btw just now saw how script started do life skills (forgot how it called correctly), while character was farming on spot.  Is it right behavior of script or some bug?
[life skill](https://imgur.com/a/GoEclEL) mean this

**Comment by hatemosphere** [comment]
this works okay on full hd and bug is present only on 1280x720

**Comment by kajdaq** [comment]
Full HD resolution being 1920x1080? or 2K - 4K?

**Comment by kajdaq** [comment]
> Btw just now saw how script started do life skills (forgot how it called correctly), while character was farming on spot. Is it right behavior of script or some bug?
> [life skill](https://imgur.com/a/GoEclEL) mean this

No.  please file as issue with log and screenshots if able to replicate.

**Comment by just4omp** [comment]
> Full HD resolution being 1920x1080? or 2K - 4K?

Full HD is 1920x1080 as i remember.

I will try to catch script on this again. Sometime difficult because its happens unpredictably

**Comment by just4omp** [comment]
Cant catch script right t the moment he start use life skills. Here is log, may be between 12:20 and 12:34 script started use it. I will try catch again, may be will use screen recorder. Now this bug was on 1920x1080
[Log](https://we.tl/t-xvUfj5B7jh)

**Comment by kajdaq** [comment]
Looks like the script is using an RGB check for the conditional according to the log.
